framework/src/modules/dpos_v2/stores/eligible_delegates.ts

```diff
@@ -30,20 +33,72 @@ export const eligibleDelegatesStoreSchema = {
 	},
 };
 
+// uint64 + address
+const KEY_LENGTH = 8 + 20;
+
 export class EligibleDelegatesStore extends BaseStore<EligibleDelegate> {
 	public schema = eligibleDelegatesStoreSchema;
 
+	private _config!: ModuleConfig;
+
+	public init(config: ModuleConfig) {
+		this._config = config;
+	}
+
 	public getKey(address: Buffer, delegateWeight: bigint): Buffer {
 		const buffer = Buffer.alloc(8);
 		buffer.writeBigUInt64BE(delegateWeight);
 		return Buffer.concat([buffer, address]);
 	}
 
 	public async getTop(context: ImmutableStoreGetter, count: number) {
-		return this.iterate(context, { limit: count, reverse: true });
+		return this.iterate(context, {
+			gte: Buffer.alloc(KEY_LENGTH, 0),
+			lte: Buffer.alloc(8 + 20, 255),
```

Comment:
You've already declared `KEY_LENGTH = 8 + 20`. You can use it both for `gte` and `lte` property.

The same goes for the usage of `iterate()` in `getAll()` method.